From 9c6ec4cd15b76cc5508b161b26fb96ae9e734be4 Mon Sep 17 00:00:00 2001
From: Joy Cho <joy.cho@hardkernel.com>
Date: Mon, 18 Jun 2018 17:41:23 +0900
Subject: [PATCH] (FIXME) Makefile: fix toolchain path

CROSS_COMPILE setting routine should be removed in uboot Makefile
and main build mk of buildroot or android will manage it.

Change-Id: I032fdcf1ad4de1f3f1863c60c15589898dec0980
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 37b94d52bd..81c9b471d7 100644
--- a/bl33/Makefile
+++ b/bl33/Makefile
@@ -246,8 +246,8 @@ ifeq ($(HOSTARCH),$(ARCH))
 CROSS_COMPILE ?=
 endif
 
-export CROSS_COMPILE=/opt/gcc-linaro-aarch64-none-elf-4.8-2013.11_linux/bin/aarch64-none-elf-
-export CROSS_COMPILE_T32=/opt/gcc-arm-none-eabi-6-2017-q2-update/bin/arm-none-eabi-
+export CROSS_COMPILE=/opt/toolchains/gcc-linaro-aarch64-none-elf-4.8-2013.11_linux/bin/aarch64-none-elf-
+export CROSS_COMPILE_T32=/opt/toolchains/gcc-arm-none-eabi-6-2017-q2-update/bin/arm-none-eabi-
 
 KCONFIG_CONFIG	?= .config
 export KCONFIG_CONFIG
-- 
2.17.1

